netmiko/yotc/yotc.py

```diff
@@ -34,12 +34,15 @@ def _send_command_more(
         cisco_wlc_ssh.py has similar function: send_command_w_enter().
         """
         prompt = self.find_prompt()
-        more_str_re = r".*--More--.*"
-        new_data = ""
-        output = ""
+        more_str_re = r"\s--More--\s*"
+        new_data = self.send_command_timing(
+            command_string, use_textfsm=False, use_genie=False, **kwargs,
+        )
+        output = new_data
```

Comment:
Why duplicate this code? Why not just have it once inside the loop?

Reply:
I use ```send_command_more```  as the default send_command func in yotc, as it have no disable_paging func, and just can expand to 1000 lines，so need use this to show all things in one time，so need enter the first command link ```show run```， if just loop the `` ``,its no place to use this func